packages/user-interface/src/pages/task/task-page.tsx

```diff
@@ -70,7 +75,6 @@ const TaskPage = () => {
 
   useEffect(() => {
     getTaskData();
```

Comment:
Deze functie kunnen we wel verwijderen als dat het enige is wat uitgevoerd word.